pontoon/teams/static/js/request.js

```diff
@@ -0,0 +1,182 @@
+/* Public functions used across different files */
+var Pontoon = (function (my) {
+  return $.extend(true, my, {
+    requestItem: {
+      /*
+       * Toggle available projects/teams and request div
+       *
+       * show Show enabled projects/teams?
+       */
+      toggleItem: function (show, type) {
+        // Toggle
+        $('.controls .request-toggle')
+          .toggleClass('back', !show)
+          .find('span')
+            .toggleClass('fa-chevron-right', show)
+            .toggleClass('fa-chevron-left', !show);
+
+        if (type === 'projects') {
+          var localeProjects = $('#server').data('locale-projects');
+          // Hide all projects
+          $('.projects')
+            .toggleClass('request', !show)
+            .find('tbody tr')
+              .toggleClass('limited', !show)
+              .toggle(!show);
+
+          // Show requested projects
+          $(localeProjects).each(function() {
+            $('.projects')
+                .find('td[data-slug="' + this + '"]')
+              .parent()
+                .toggleClass('limited', show)
+                .toggle(show);
+          });
+          Pontoon.requestItem.toggleButton(!show, 'projects');
+        } else if (type === 'team') {
+          // Hide all teams
+          $('.team-list').toggle(show);
+          // Show team form
+          $('#request-team-form').toggle(!show);
+          Pontoon.requestItem.toggleButton(!show, 'team');
+        }
+
+        $('.controls input[type=search]:visible').trigger('input');
+      },
+
+      toggleButton: function (condition, type) {
+        condition = condition || true;
+        var show = condition
+        if (type === 'projects') {
+          show = condition && $('.projects td.check.enabled:visible').length > 0;
+        } else if (type === 'team') {
+          show = condition &&
+          ($.trim($('#request-team-form #id_name').val()) !== '') &&
+          ($.trim($('#request-team-form #id_code').val()) !== '');
+        }
+
+        $('#request-item-note').toggle(show);
+        $('#request-item').toggle(show);
+      },
+
+      requestProjects: function(locale, projects) {
+        $.ajax({
+          url: '/' + locale + '/request/',
+          type: 'POST',
+          data: {
+            csrfmiddlewaretoken: $('#server').data('csrf'),
+            projects: projects
```

Comment:
Nit: You can add comma at the end of the line. That will make diff shorter in case we'll add more parameters in the future.